components/avatar.css

```diff
@@ -0,0 +1,42 @@
+.avatar{
+    border-radius: 50%;
+    
+}
+
+.avatar-xl{
+    width: 10rem;
+    height: 10rem;
+font-size: 3rem;
+}
+.avatar-l{
+    width: 8rem;
+    height: 8rem;
+    font-size:2.5rem;
+}
+
+
+.avatar-m{
+    width: 6.25rem;
+    height: 6.25rem;
+    font-size:2rem ;
+}
+
+.avatar-s{
+    width: 3rem;
+    height: 3rem;
+    font-size: 1rem;
+}
+.avatar-xs{
+    width: 2rem;
+    height: 2rem;
+    font-size:0.8rem ;
+}
+
+.avatar-text
+{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--primary-voilet);
+    color:white
```

Comment:
use color variables , it will help you in future modularity of the project. eg --COLOR-BLACK.